tests/test_syntax/test_coroutine/test_sdram.py

```diff
@@ -138,7 +135,7 @@ def init(self, refresh_cnt: m.UInt[10], rd_enable: m.Bit, wr_enable: m.Bit) -> (
             self.yield_state = INIT_REF1
             yield self.yield_state.prev(), self.command.prev()
             # for _ in range(7, -1, -1):
-            self.i = 7
+            self.i = m.uint(7, 4)
```

Comment:
I'm guessing with https://github.com/phanrahan/magma/pull/761, we'll be able to revert these changes? Not a big deal, but would make it a little easier to read/write

Reply:
yes exactly